source/guides/guides/environment-variables.md

```diff
@@ -211,7 +211,29 @@ Cypress.env('api_server') // 'http://localhost:8888/api/v1/'
 
 ## Option #5: Plugins
 
-Instead of setting environment variables in a file, you can use plugins to dynamically set them with Node code. This enables you to do things like use `fs` and read off configuration values and dynamically change them.
+Instead of setting environment variables in a file, you can use plugins to dynamically set them with Node code. This enables you to do things like use `fs` and read off configuration values and dynamically change them. For example, if you use {% url dotenv https://github.com/motdotla/dotenv#readme %} package to read the `.env` file, you could then grab the needed environment variables from the `process.env` object and place them into `config.env` to make available in the tests:
```

Comment:
```suggestion
Instead of setting environment variables in a file, you can use plugins to dynamically set them with Node code. This enables you to do things like use `fs` and read off configuration values and dynamically change them. 

For example, if you use the {% url dotenv https://github.com/motdotla/dotenv#readme %} package to read the `.env` file, you could then grab the needed environment variables from the `process.env` object and place them into `config.env` to make available in the tests:
```